src/docGeneratorMain.ts

```diff
@@ -202,6 +202,12 @@ async function main() {
     "lodash-fp"
   )
 
+  console.log("Generate lwc description files")
+  await docGenerator.downloadDocs(
+    `${githubBaseUrl}/salesforce/eslint-plugin-lwc/master/docs/rules/`,
+    "@lwc"
```

Comment:
This should be:
@lwc/lwc
And should be placed alphabetically with the remaining plugins starting with "@". That is also how it orders when adding to package.json and we should be consistent.